single-sign-on/requirements/index.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Single Sign-on requirements
+keywords: Single Sign-on, SSO, sign-on, requirements
+title: Requirements
+---
+
+## Prerequisites
+
+* You must first notify your company about the new SSO login procedures
+* Verify that your org members have Docker Desktop version 4.4.2 installed on their machines
```

Comment:
The FAQs specify `4.4.0 or later`.
Suggest aligning whichever version is correct and use `or later` wording here.

Reply:
Updated 4.13.0 for both here and FAQs